src/Jellyfin.Database/Jellyfin.Database.Providers.Sqlite/SqliteDatabaseProvider.cs

```diff
@@ -120,4 +126,25 @@ public Task RestoreBackupFast(string key, CancellationToken cancellationToken)
         File.Copy(backupFile, path, true);
         return Task.CompletedTask;
     }
+
+    /// <inheritdoc/>
+    public async Task PurgeDatabase(JellyfinDbContext dbContext, IEnumerable<string>? tableNames)
+    {
+        ArgumentNullException.ThrowIfNull(tableNames);
+
+        var deleteQueries = new List<string>();
+        foreach (var tableName in tableNames)
+        {
+            deleteQueries.Add($"DELETE FROM \"{tableName}\";");
```

Comment:
Can we use `TRUNCATE` here?

Reply:
When not specifying a filter or having triggers, sqlite does that on its own

https://sqlite.org/lang_delete.html
4.